axum/CHANGELOG.md

```diff
@@ -9,14 +9,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - None.
 
+# 0.5.3 (19. April, 2022)
+
+- **fixed:** Revert [#924] which was an accidental breaking change.
+
 # 0.5.2 (19. April, 2022)
 
 - **added:** Add `AppendHeaders` for appending headers to a response rather than overriding them ([#927])
 - **added:** Add `axum::extract::multipart::Field::chunk` method for streaming a single chunk from
   the field ([#901])
+- **changed:** Allow `Error: Into<Infallible>` for `Route::{layer, route_layer}` ([#924])
 - **fixed:** Fix trailing slash redirection with query parameters ([#936])
 
 [#901]: https://github.com/tokio-rs/axum/pull/901
+[#924]: https://github.com/tokio-rs/axum/pull/924
 [#927]: https://github.com/tokio-rs/axum/pull/927
 [#936]: https://github.com/tokio-rs/axum/pull/936
```

Comment:
How about

```suggestion
# 0.5.3 (19. April, 2022)

- **added:** Add `AppendHeaders` for appending headers to a response rather than overriding them ([#927])
- **added:** Add `axum::extract::multipart::Field::chunk` method for streaming a single chunk from
  the field ([#901])
- **fixed:** Fix trailing slash redirection with query parameters ([#936])

[#901]: https://github.com/tokio-rs/axum/pull/901
[#927]: https://github.com/tokio-rs/axum/pull/927
[#936]: https://github.com/tokio-rs/axum/pull/936

# 0.5.2 (19. April, 2022)

Yanked, as it contained an accidental breaking change.
```

Reply:
I like it!